studio/lib/common/fetch/head.ts

```diff
@@ -24,3 +24,32 @@ export async function head<T = any>(
     return handleError(error, requestId)
   }
 }
+
+export async function headWithTimeout<T = any>(
```

Comment:
@phamhieu thinking maybe we can merge head and headWithTimeout into one by passing an optional timeout parameter? could possible do the same for get, post, _delete as well (maybe over engineering at this point)